api/src/main/java/org/openmrs/module/reporting/cohort/definition/DrugOrderCohortDefinition.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
Are you using the latest license version?

Reply:
updated